src/components/ManualWorkFlowDashboard.jsx

```diff
@@ -0,0 +1,35 @@
+import { useNavigate } from 'react-router-dom';
+import CloseLogo from "../logos/CloseLogo.svg";
+
+function ManualWorkFlowDashboard() {
+    let navigate = useNavigate();
+
+    return (
+       <div className='bg-bg h-screen bg-bg'>
+        <button onClick={()=> navigate(-1)} className="text-sm ml-12 px-1 py-1 font-bold" > 
+         <span>{`<`}Back</span>
```

Comment:
All these hardcoded strings should be passed as props.

Reply:
addressed